target/linux/mediatek/filogic/base-files/etc/hotplug.d/firmware/11-mt76-caldata

```diff
@@ -45,10 +45,15 @@ case "$FIRMWARE" in
 	;;
 "mediatek/mt7986_eeprom_mt7976_dbdc.bin")
 	case "$board" in
-	asus,tuf-ax4200)
+	asus,rt-ax59u|\
+	asus,tuf-ax4200|\
+	asus,tuf-ax6000)
```

Comment:
Did you intentionally add the `asus,tuf-ax6000` here?

Reply:
It was an copy&paste error, sorry